src/pages/movieList.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import {createMovieCard} from '../components/card.js';
+import {renderPageLayout} from '../layouts/createLayout.js';
+
+export function createMovieList() {
```

Comment:
@Mmarvinn Task wasn't about MovieList, why did you put it in this PR?

Reply:
sorry, my bad, in next time i didn`t will be put other files in PR